Components/Text/Text.js

```diff
@@ -6,7 +6,7 @@ const baseStyles = css`
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
   font-style: normal;
-  color: ${getColorValue('black')};
+  color: ${getColorValue('white')};
```

Comment:
might be nice to also add a subtle shadow to the text like did to div shapes, but need to keep in mind a11y.